EventFilter/CSCRawToDigi/src/CSCDigiToRaw.cc

```diff
@@ -324,18 +324,32 @@ void CSCDigiToRaw::add(const CSCCorrelatedLCTDigiCollection& corrLCTDigis, FindE
   }
 }
 
+void CSCDigiToRaw::add(const GEMPadDigiClusterCollection& gemPadClusters, FindEventDataInfo& fedInfo) const {
+  for (auto j = gemPadClusters.begin(); j != gemPadClusters.end(); ++j) {
```

Comment:
range based for can be better here

Reply:
It that available on MuonDigiCollections?

Reply:
It is C++, not really dependent on the class...

```C++
  for (auto & jclus : gemPadClusters) {
    const GEMDetId& gemDetId = jclus.first;
    ...
  }
```

Reply:
Done.